src/components/project/moreProjects.tsx

```diff
@@ -1 +1,196 @@
-export {};
+import { Projects } from "../../interfaces/project";
+import { IconButton } from "../global/iconButton";
+import styled from "@emotion/styled";
+import { breakpoints, dimensions } from "../../styles/variables";
+import { getRemSize } from "../../styles/globalCss";
+import useIsDesktop from "../../hooks/useIsDesktop";
+import { Fragment } from "react";
+import Image from "next/image";
+
+interface IProjectsData {
+  projects: Projects;
+}
+
+const StyledHeading = styled.h2`
+  font-weight: 500;
```

Comment:
I know it says a font weight of 500 on Figma, but the title looks a bit lighter to me. Consider changing it to 400